### PR TITLE
Slow down YouTube video payback speed

### PR DESCRIPTION
**Pull request by omar2205**
sloooooow video playback speeeeeed

### CODE REVIEW

**Comment by omar2205** [comment]
more ideas:

1. random speed value
2. keep re-applying the speed, so they don't change it in the player

File: extension/plugins/content/SlowYTVids.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import ContentPlugin from '../../lib/ContentPlugin.js';
+
+class PageBlur extends ContentPlugin {
```
Comment (w3cj) [inline extension/plugins/content/SlowYTVids.js:3]: Rename class to match

**Review by w3cj — CHANGES_REQUESTED** [review]

File: extension/plugins/content/SlowYTVids.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    enabled: false,
+    // TODO: Maybe enable to all websites?
+    yt_only: true,
+    speed: .4,
+  }
+
+  run() {
+    const slowVID = () => {
+      document.querySelectorAll('video').forEach(vid => {
+        vid.playbackRate = (typeof speed === 'undefined') ? .4 : speed;
```
Comment (w3cj) [inline extension/plugins/content/SlowYTVids.js:20]: I think you should use `this.settings.speed` here instead of just `speed`

**Review by omar2205 — COMMENTED** [review]
forgot that, duh

File: extension/plugins/content/SlowYTVids.js (hunk 2 above)
Comment (omar2205) [reply extension/plugins/content/SlowYTVids.js:20]: yeah, you're right.

**Pull request merged**